examples/Developing_hallucination_guardrails.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Developing Hallucination Guardrails\n",
+    "\n",
+    "In this notebook we'll walk through the process of developing a guardrail that checks model outputs and against hallucinations. \n",
```

Comment:
can we briefly explain what a guardrail is and/or link to the existing cookbook: https://cookbook.openai.com/examples/how_to_use_guardrails?